seatunnel-connectors-v2/connector-cassandra/src/main/java/org/apache/seatunnel/connectors/seatunnel/cassandra/sink/CassandraSink.java

```diff
@@ -106,8 +107,11 @@ public void prepare(Config pluginConfig) throws PrepareFailException {
             throw new CassandraConnectorException(
                     SeaTunnelAPIErrorCode.CONFIG_VALIDATION_FAILED,
                     String.format(
-                            "PluginName: %s, PluginType: %s, Message: %s",
-                            getPluginName(), PluginType.SINK, checkResult.getMsg()));
+                            "PluginName: %s, PluginType: %s, Message: %s %s",
+                            getPluginName(),
+                            PluginType.SINK,
+                            checkResult.getMsg(),
+                            ExceptionUtils.getMessage(e)));
```

Comment:
```suggestion
                            "PluginName: %s, PluginType: %s, Message: %s",
                            getPluginName(),
                            PluginType.SINK,
                            ExceptionUtils.getMessage(e)));
```

Reply:
checkResult useless in here. Because if checkResult is failed, it never reach here.